src/ploneintranet/search/results.py

```diff
@@ -1,3 +1,69 @@
 """
 Implementations of the ISearchResult and ISearchResponse
 """
+from zope.interface import implements
+from zope.globalrequest import getRequest
+
+from .interfaces import ISearchResult, ISearchResponse
+
+
+class SearchResult(object):
+    """
+    Data structure containing parsed results from search backend
+    """
+    implements(ISearchResult)
+
+    #: The title of the indexed document (str)
```

Comment:
These comments seem superfluous, considering they are just repeating what is in the interface.